### ISSUE TITLE
"nest build" doesn't terminate, stuck on "Entrypoint main"

### ISSUE THREAD

**Issue by jacobgoh101** [nestjs/nest-cli#958]
## I'm submitting a...

<pre><code>
[ ] Regression 
[x] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

In [this NestJS monorepo project](https://github.com/jacobgoh101/nestjs-build-problem-demo), 

when I ran `yarn build` or `yarn build mobile-api`, this is the result:

```sh
yarn run v1.21.1
$ rimraf dist
$ nest build
Starting type checking service...
Hash: 5d229137b12354a1fc13
Version: webpack 4.42.0
Time: 6761ms
Built at: 11/20/2020 11:13:11 AM
Entrypoint main = apps/mobile-api/main.js

# STUCK FOREVER HERE .....
```

## Expected behavior

`nest build` is expected to terminate automatically after the build is successful.

## Minimal reproduction of the problem with instructions

It can be reproduced in [this NestJS monorepo project](https://github.com/jacobgoh101/nestjs-build-problem-demo), 

## What is the motivation / use case for changing the behavior?

N/A

## Environment

```


[System Information]
OS Version     : macOS Catalina
NodeJS Version : v10.21.0
YARN Version    : 1.21.1 

[Nest CLI]
Nest CLI Version : 7.0.0 

[Nest Platform Information]
platform-express version : 7.0.0
serve-static version     : 2.1.3
passport version         : 7.1.0
swagger version          : 4.6.1
common version           : 7.0.0
config version           : 0.5.0
bull version             : 0.3.0
core version             : 7.0.0
```

**Comment by jacobgoh101** [comment]
Based on https://github.com/nestjs/nest-cli/issues/558, it seems like the reason might be Webpack HMR. 

I have tried to turn off or remove all config that seems related to HMR or watch mode, but the problem still persists.

**Comment by jacobgoh101** [comment]
fixed after updating `@nestjs/cli` to `7.5.3` ...
